app/src/main/java/org/p2p/wallet/history/ui/token/adapter/HistoryAdapter.kt

```diff
@@ -44,8 +44,7 @@ class HistoryAdapter(
             notifyDataSetChanged()
         } else {
             val oldItems = ArrayList(currentItems)
-            currentItems.clear()
-            currentItems += newTransactions.mapToItems()
+            currentItems = newTransactions.mapToItems().toMutableList()
```

Comment:
what is the difference?

Reply:
it's same =) but in one operation

Reply:
It's not the same.

`currentItems +=` means that the list adds the other collection. 

`toMutableList` produces the new list.

Additionally, we are assigning a new reference and the old list is now should be removed. 
It's a redundant operation for GC